src/_includes/layouts/base.njk

```diff
@@ -22,7 +22,11 @@
     <link rel="stylesheet" type="text/css" href="{{ '/assets/fonts/inter/600.css' | url }}">
     <link rel="stylesheet" type="text/css" href="{{ '/assets/fonts/inter/700.css' | url }}">
     <link rel="stylesheet" href="{{ '/assets/forkawesome/css/fork-awesome.min.css' | url }}">
-    <!-- incredibly based web app to make mod lists at https://moddermore.vercel.app -->
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand&display=swap" rel="stylesheet">
+    <link href="https://unpkg.com/pretendard@1.3.6/dist/web/static/pretendard.css" rel="stylesheet">
```

Comment:
is this font open-source ?

Reply:
The font should be hosted locally in the repository.

Reply:
seems doable for the Quicksand font, but idk for pretendard, which appears to be an adobe font

Reply:
Pretendard is open source
https://github.com/orioncactus/pretendard

Reply:
> The font should be hosted locally in the repository to avoid a connection to Google servers (privacy concern).

I was planning on doing this soon, likely after demos for https://discord.com/channels/1031648380885147709/1031934924401422376/1032381116209975437 are finished

Reply:
May I suggest https://outfit.io/font and https://fonts.google.com/specimen/Lexend ?
Both are licensed under the SIL OFL IIRC.
Note on Lexend: If you search Lexend on Google Fonts, there are a million width options. Just thought you should know.